apps/dashboard/src/interpret-vision/__mock_data__/visionData.ts

```diff
@@ -1,115 +1,1080 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { IVisionExplanationDashboardData } from "@responsible-ai/core-ui";
+import { DatasetTaskType, IDataset } from "@responsible-ai/core-ui";
 
 import { images } from "./images";
 
-export const visionData: IVisionExplanationDashboardData = {
-  classNames: ["tench"],
-  images,
-  predictedY: [
-    "cocktail_shaker",
-    "spindle",
+export const visionData: IDataset = {
+  categorical_features: [],
+  class_names: [
```

Comment:
Since there are a max of 100 unique class_names that can be pointed to by the 50 predy's or 50 truey's, I'm going to try trimming these down to only 100.